stories/smart-sort.stories.ts

```diff
@@ -0,0 +1,40 @@
+import algoliasearch from 'algoliasearch/lite';
+import { storiesOf } from '@storybook/html';
+import { withHits } from '../.storybook/decorators';
+import smartSort from '../src/widgets/smart-sort/smart-sort';
+
+const searchOptions = {
+  appId: 'C7RIRJRYR9',
+  apiKey: '77af6d5ffb27caa5ff4937099fcb92e8',
+  indexName: 'test_Bestbuy_vr_price_asc',
+  searchClient: algoliasearch('C7RIRJRYR9', '77af6d5ffb27caa5ff4937099fcb92e8'),
```

Comment:
searchClient gets created from appId, apiKey options iirc, no need to create the search client again since there's no special arguments to algoliasearch anymore

Reply:
You're right. I used to have some argument to override the host to a specific one, but forgot to delete this https://github.com/algolia/instantsearch.js/pull/4648/commits/f681b8778dc1f9f90c8c3e0c1fc17425c7b28269